src/main/java/carbonconfiglib/networking/snyc/BulkSyncPacket.java

```diff
@@ -71,7 +71,7 @@ public void process(Player player) {
 			result = ReloadMode.or(result, packet.processEntry(player));
 		}
 		if(result != null) {
-			player.sendSystemMessage(result.getMessage());
+			player.sendMessage(result.getMessage(), player.getUUID());
```

Comment:
Please use Util.NillUUID or whatever it is called. There is no sender UUID, this is not a normal chat message but a system one.